pkg/trace/obfuscate/sql.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const sqlQueryTag = "sql.query"
-const nonParsableResource = "Non-parsable SQL query"
+const nonParsableResource = "Cannot obfuscate: Non-parsable SQL query"
```

Comment:
I agree with you that this is more explicit but unfortunately making this change would break backwards compatibility for anyone tracking this value, which is very likely. I'm afraid we can't make this change. At most we could edit the log output and make that better, but I'm not sure if that'll actually help.